openedx/core/lib/api/tests/test_authentication.py

```diff
@@ -43,30 +48,39 @@ def put(self, request):  # pylint: disable=missing-docstring,unused-argument
 # This is the a change we've made from the django-rest-framework-oauth version
 # of these tests.  We're subclassing our custom OAuth2AuthenticationAllowInactiveUser
 # instead of OAuth2Authentication.
-class OAuth2AuthenticationDebug(OAuth2AuthenticationAllowInactiveUser):  # pylint: disable=missing-docstring
+class OAuth2AuthenticationDebug(authentication.OAuth2AuthenticationAllowInactiveUser):  # pylint: disable=missing-docstring
     allow_query_params_token = True
 
 
+TokenError = namedtuple('TokenError', ['token', 'error_code'])
+ScopeStatuses = namedtuple('ScopeStatuses', ['scope', 'read_status', 'write_status'])
```

Comment:
Is there a reason these are declared away from where they are actually used?  I needed to scroll up here to determine the order of the fields.  You can still declare them within the Test class; you'll just have to prefix them with `self.`.